### PR TITLE
Use Python 3.12 as base for testing

### PR DESCRIPTION
**Pull request by mdomke**

### CODE REVIEW

File: schwifty/registry.py [hunk 1]
```diff
@@ -11,8 +11,8 @@
 
 
 try:
-    from importlib.abc import Traversable
     from importlib.resources import files
+    from importlib.resources.abc import Traversable
 except ImportError:
     from importlib_resources import files  # type: ignore
     from importlib_resources.abc import Traversable  # type: ignore
```
Comment (Natim) [inline schwifty/registry.py:18]:
```suggestion
try:
    from importlib_resources import files
    from importlib_resources.abc import Traversable
except ImportError:
    from importlib.abc import Traversable  # type: ignore
    from importlib.resources import files  # type: ignore
```

Comment (mdomke) [reply schwifty/registry.py:18]: This would however not remove the `DeprecationWarning` that was complained about in #158, right?

Comment (Natim) [reply schwifty/registry.py:18]: Maybe I was too quick indeed. What I wanted to say is that we should try to import the Python 3.12 way, then the Python 3.9/3.10 and finally 3.8

Comment (Natim) [reply schwifty/registry.py:18]: For reference the error with 3.9 and 3.10 is:

```
ModuleNotFoundError: No module named 'importlib_resources'
```

**Pull request merged**